app/controllers/users_controller.rb

```diff
@@ -0,0 +1,34 @@
+class UsersController < ApplicationController
+
+  def new
+  end
+
+  def show
+    @user = User.find(session[:user_id])
```

Comment:
we might think about adding a sad path here if a user is not signed in. For example: render file: '/public/401' unless current_user

Reply:
Good idea Dani; I'll add to refactor wishlist